docs/guides/meta/meta-avatars.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: docs
+title: Meta Avatars
+---
+import metaPlatform from './meta-avatars/install-meta-platform.mp4'
+
+# Meta Avatars
+Meta / Oculus come with rich prebuilt avatars that work out of the box with Normcore. This guide will demonstrate how to use the built-in **MetaAvatar** component to set up networked Meta Avatars with voice chat in a few minutes.
+
+TODO (Stephen): I'd love a video here of two avatars waving to the camera in the space. If you send me a Quest build, I can hop in and wave while you screen record the editor window. We'll want to make sure the Game view is full screen so the resolution is crisp.
+
+## Install the Normcore Meta package
+Before we start, make sure you've [set up Normcore](../../essentials/getting-started.md) and have configured all of [Meta's platform prerequisites](./meta-platform-prerequisites.md).
+
+Once Normcore is set up in your project, you'll be able to install the Normcore Meta UPM package. Open up Package Manager, switch to **My Registries**, and install **Normcore Meta Platform** from the Normal package registry.
+
+<video width="100%" controls><source src={metaPlatform} /></video> 
+
+## Set up Meta Avatars
+Import the VR examples (TODO: Are we including this in VR Examples or does the Normcore Meta package have its own samples?). Drag the "Realtime + Meta Avatar Player" into your scene.
```

Comment:
@stephen-radial I'm trying the guide out now, and I realize I forgot the step where we drag in the components that are required in the scene like `OvrAvatarManager`. I'd like to do that step first thing in this section. Which also means the rest of the videos in this section need to show that prefab in the Hierarchy panel.

Reply:
Given that the only prefab that has OvrAvatarManager on it is in the Meta Avatar samples, I'd love to just bundle the prefab itself in our package (an ideally give it a new GUID so it doesn't conflict)

Reply:
It looks like their `OvrAvatarManagerBase` prefab isn't even enough smh lol. The `MetaAvatarSdkManager` + `MetaAvatarShaderManager` combo you've been copying out of their SimpleScene is what we should bundle as its own prefab in our package. Ideally without any child prefab references. It should reference the scripts directly.